src/components/schemas/statement_mappings.yaml

```diff
@@ -0,0 +1,81 @@
+suggest_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
```

Comment:
`org_id` column is also missing

Reply:
Not needed for this api sir, as this is a suggestion api.